hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAccountWipeHandlerTest.java

```diff
@@ -631,7 +632,7 @@ void nftSerialNumsIsEmpty() {
 
             assertThatThrownBy(() -> subject.handle(context))
                     .isInstanceOf(HandleException.class)
-                    .has(responseCode(INVALID_WIPING_AMOUNT));
+                    .has(responseCode(FAIL_INVALID));
```

Comment:
Why did you change the test response code here?

Reply:
It's expected in the HAPI test to fail for this with FAIL_INVALID. And with the changes in the handler this is how it fails. Can't say for sure what's the correct error, but I treat the HAPI Tests as source of truth 😄

Reply:
This is just unit test. Np.